client/lib/billing-card/app/controllers/overlays/billing-card.js

```diff
@@ -0,0 +1,131 @@
+import Ember from 'ember';
```

Comment:
Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.<br>'Ember' is not defined.<br>'from' is not defined.